cypress/e2e/auth/signup.cy.js

```diff
@@ -0,0 +1,24 @@
+describe('Signup test', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000/sign-up');
+  });
+
+  it.only('Login should display error message', () => {
+    cy.get('#username').type('KevinLu');
+    cy.get('#email').type('turnkevin@gmail.com');
```

Comment:
Better to fetch the credentials from an external JSON file

Reply:
I have already add this to external JSON file